json/test_mir_ion.d

```diff
@@ -0,0 +1,32 @@
+/+ dub.sdl:
+name "test_mir_ion"
+dependency "mir-ion" version="~>0.1.11"
+dependency "mir_common_json" path="mir-common-json"
+dflags "-mcpu=native" "-linkonce-templates" "-enable-cross-module-inlining" platform="ldc"
+lflags "-L../common/libnotify/target/"
++/
+import mir_common_json;
+import mir.ion.deser.json: deserializeJson;
+import std.file: read;
+
+auto calc(string text) @nogc @safe pure
```

Comment:
I not sure, is nogc correct here? Deserialize should use it

Reply:
It is correct. Mir support `@nogc` deserialization if underlying types ara `@nogc` by construction. For example, Mir ref-counted arrays or just a `@nogc` output range like `CoordinateAvg`.